src/routes/register.rs

```diff
@@ -0,0 +1,24 @@
+use rocket::{post, request::Form, State};
+use rocket_contrib::database;
+use rocket_jwt::{JWT, RocketJWT};
+
+#[database("sqlite_database")]
+pub struct DbConn(diesel::SqliteConnection);
+
+#[derive(FromForm)]
+struct RegisterForm {
+    ip_address: String,
```

Comment:
no need to send ip address in register form

Reply:
i think register endpoint should only be accessed locally then right?